api/src/main/java/org/openmrs/module/quarterlyreporting/regimenhistory/RegimenUtils.java

```diff
@@ -146,7 +147,7 @@ public static void setRegimen(Patient patient, Date effectiveDate, Collection<Dr
                                     } else {//new order has infinite end date
                                         if (!regimenComponentIsTheSameAsDrugOrderExcludingDates(before, newOrder)){
                                           //(OUTCOME6)
-                                                newOrder.setDiscontinuedDate(before.getStartDate());

```

Comment:
For all the changes to this class, refer to the same file upgraded attached above